### DISCUSSION TITLE
`nix build .` doesn't work in Ema example

### DISCUSSION THREAD

**Discussion by Emmanuel-PLF** [srid/haskell-flake#17] [General]
I use this example https://github.com/srid/ema/blob/master/flake.nix with GHC8107 (nixpkgs unstable and I can't build the package, with nix build . or nix run . I've got this error : 
flake 'git+file:///home/manu/build/hsnix/mong-servant' does not provide attribute 'packages.x86_64-linux.default' or 'defaultPackage.x86_64-linux'. Have you got an idea ? 
Thanks

**Comment by srid** [comment]
There is no default package in that one (only a default shell); you can try `nix build .#ghc90` or `nix build .#ghc92`.
